flterm.c

```diff
@@ -694,6 +706,8 @@ static void do_terminal(
 	fds[1].fd = serialfd;
 	fds[1].events = POLLIN;
 
+    signal(SIGINT,sig_func); //Install the signal handler
```

Comment:
Probably want to add this a lot earlier?

Reply:
I put it just above the loop in case there was some need for default ^c breaking out of the transfer. 

not sure it matters.